mqt/syrec/syrec_editor.py

```diff
@@ -97,7 +97,9 @@ def load(self, circ):
                 )
             )
             self.outputs.append(
-                self.add_line_label(width, i * 30, circ.outputs[i], QtCore.Qt.AlignmentFlag.AlignLeft, circ.garbage[i])
+                self.add_line_label(
+                    width, i * 30, circ.outputs[i], QtCore.Qt.AlignmentFlag.AlignLeft, circ.constants[i] is not None
```

Comment:
What does it mean for a label to be highlighted red?
In my opinion labels at the beginning of the circuit should be red if they are constant, while labels at the end should be red if they are garbage. but I might be wrong in that regard.

Reply:
> One more thing: Please create a separate feature branch in your fork of the repository instead of working on the main branch in your fork in the future. If you don't do that, it gets really hard to keep your fork in sync with the main repository.

One more thing, this PR can be closed right?